golden_data.Rmd

```diff
@@ -1,56 +1,90 @@
+![](images/UniProt_pmo.png){width=100%}
+To remove these off-target search results, we will need to expand our search terms and for this we will turn to the "Advanced" drop-down menu on the right side of the search bar. Add "pmoa" for the *Gene name [GN]* and remove all sequences with unresolved lineages by specifying "uncultured bacterium" for *Taxonomy [OC]* with the *NOT* logical operator. The latter filter was applied since these poorly annotated sequences are typically derived from short environmental samples and are not taxonomically informative. The advanced search fields should look identical to the figure below.
 
-![](images/EggNOG_xmoa_1.png){width=100%}
+![](images/pmoA_advanced_search.png){width=100%}
 
-Click "Download" at the bottom-left of the panel and select "All _XX_ sequences (FASTA)", where _XX_ is the number of sequences belonging to the orthologous group (OG). This will open a new window with all sequences from that OG in FASTA format. Download this file to a file on your computer following the format &lt;OG name&gt;_EggNOGv5.faa where &lt;OG name&gt; is the EggNOG ortholog identifier.
+This more specific search should yield 3882 entries with the vast majority being predicted pmoA sequences.
 
-Repeat the above steps with the other EggNOG identifier.
+#### Download sequences
+
+Downloading sequences in UniProt is quick and easy. Click the ![](images/uniprot_dl_icon.png){width=50} button (atop the table), ensure the format is FASTA, and click "Go" to specify where the file should be saved on your computer. Change the file name to `UniProt_PmoA.fasta`.
+
+#### Download taxonomic lineages
+
+UniProt sequence names are not searchable within the NCBI's database, so software (including TreeSAPP) is unable to pair them with their corresponding taxonomic lineages like with EggNOG. Therefore, a separate table must be downloaded from UniProt and provided to TreeSAPP during reference package creation.
+
+To create a properly formatted table, we must first change the columns in the UniProt table of search results. Begin by clicking the ![](images/uniprot_cols_icon.png){width=50} icon to the right of the "Download" button.
+
+![](images/uniprot_columns.png){width=100%}
+
+Once there, only check the boxes for organism, superkingdom, phylum, class, order, family, genus and species as shown above.
+Click "Save" near the top-right of the page to save your selection.
+
+You can export this table similar to before.
+Click the ![](images/uniprot_dl_icon.png){width=50} button to select all sequences, change the format to "Tab-separated", then click "Go" to save the table to your computer.
+Change the file name to `UniProt_PmoA.tab`.
+
+To gather all sequences in UniProt for the XmoA reference package, repeat the previous two steps with the string "Ammonia monooxygenase alpha subunit" and gene name "amoA".
```

Comment:
Minor
Depending on how literally this "repeat above" instruction is followed, you can get anywhere from 1 to a few hundred results, which may be fine

Reply:
Hmm, yeah it is a bit vague. I can see where this might go wrong. Thanks for flagging!